liveOnReboot/liveOn/View/Calendar/UpcomingEventsView.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  UpcomingEventsView.swift
+//  liveOnReboot
+//
+//  Created by Keum MinSeok on 2022/07/17.
+//
+
+import SwiftUI
+// Test용으로 넣어 놓은거고 UI에 맞게 수정할 예정
+struct UpcomingEventsView: View {
+    let event : UpcomingEventsModel
```

Comment:
세미콜론은 뒤에만 띄어쓰기를 쓰는게 일반적이에요!

```suggestion
    let event: UpcomingEventsModel
```

Reply:
오늘도 한 가지 배웠습니다 밀 센세..